### PR TITLE
Added Tooltippery and DiscordVR to manifest

### PR DESCRIPTION
**Pull request by Psychpsyo**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3593,6 +3593,70 @@
...
+                            "sha256": "aa12abbb22fd9b5f1a87c79e8b1f62c6baed1433940039cffe4f1d65130cc9e0"
+                        }
+                    ]
+                }
+            }
+        },
+        "Psychpsyo.DiscordVR": { 
+            "name": "DiscordVR",
+            "description": "Adds a Discord tab to your dash, next to the contacts tab.",
+            "category": "Dash Tweaks",
```
Comment (EIA485) [inline manifest.json:3632]: meme would probably fit better here

**Review by EIA485 — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 1 above)
Comment (Psychpsyo) [reply manifest.json]: ~~Excuse me, but the category is called 'Memes', not 'meme'.~~ Yeah, you're right.

**Comment by XDelta** [comment]
DiscordVR:
`AssemblyFileVersion` and `AssemblyVersion` are both set to 1.0.0 / 1.0.0.0
NeosMod.Version is set to 1.0.0

**Comment by Psychpsyo** [comment]
> DiscordVR 1.0.1: `AssemblyFileVersion` and `AssemblyVersion` are both set to 1.0.0 / 1.0.0.0 NeosMod.Version is set to 1.0.0

The thing is now fixed.

**Review by XDelta — APPROVED** [review]

**Pull request merged**